mithril.js

```diff
@@ -28,6 +28,19 @@
 		return "v0.2.5"
 	}
 
+	var div = document.createElement("div")
+	var frag = document.createDocumentFragment()
+
+	function toDOM(HTML){
+		div.innerHTML = HTML
```

Comment:
I'd expect a runtime error in that case — mind you it's so hard to work out what's actually going on with the old test suite, I'm not sure I'd know for certain if there was. Worth a look for certain.